pallets/collator-selection/src/tests.rs

```diff
@@ -34,16 +34,16 @@ use crate as collator_selection;
 use crate::{mock::*, Error};
 use frame_support::{
 	assert_noop, assert_ok,
-	traits::{Currency, GenesisBuild, OnInitialize},
+	traits::{fungible, GenesisBuild, OnInitialize},
```

Comment:
В тестах вроде бы используется только `fungible::Mutate`. Может явно его здесь указать? 
```suggestion
	traits::{fungible::Mutate as MutateFungible, GenesisBuild, OnInitialize},
```

Reply:
А какая разница, fungible::Mutate, или MutateFungible? Мне кажется что первый вариант лучше

Reply:
Parity у себя повсюду пишет fungible::Mutate. Имхо, лучше сделать по аналогии

Reply:
Квалифицированное большинство за вариант парити, резолвлю)